fairscale/experimental/wgit/signal_sparsity.py

```diff
@@ -112,6 +155,70 @@ def sst_dst_to_dense(self, sst: Tensor, dst: Tensor = None) -> Tensor:
         """
         pass
 
+    def _transform(self, dense: Tensor) -> Tensor:
+        """
+        Applies the transformation algorithm chosen by the caller to the input dense tensor.
+        Returns the transformed tensor.
```

Comment:
move this into a "Returns:" section. Check other code in repo.py or sha1_store.py see what it may look like.

Also, add info like: transformed tensor with same shape/dtype as the input tensor, only transformed into frequency domain. Comments should be useful to the reader, should not state what's already obvious from the code.